### PR TITLE
Database Schema and Queries for Vets, Specializations, and Visits

### PR DESCRIPTION
**Pull request by MarcoDDM**
This pull request introduces the following changes:

- Create a table named "vets" with columns "id" (autoincremented integer, primary key), "name" (string), "age" (integer), and "date_of_graduation" (date).
- Create a "join table" named "specializations" to handle the many-to-many relationship between "species" and "vets".
- Create a "join table" named "visits" to handle the many-to-many relationship between "animals" and "vets", including the date of the visit.
- Insert data for vets, specialties, and visits based on the provided information.
- Implement queries to answer the provided questions about the database.
- Include a screenshot of the query results in the pull request description.

join_tables2

These changes enhance the database structure and provide the necessary functionality to manage the relationships between species, vets, and animals. The included screenshot showcases the query results, giving a visual representation of the data. Please review and provide any feedback or suggestions for improvement.

### CODE REVIEW

File: queries.sql
Comment (Kamba56) [inline queries.sql:1]: - [x] Great job adding queries to query,sql file. Kindly update the schema.sql file and data.sql file too. All queries that create or change tables should be added to the schema.sql file and all commands that add data to the tables should be added to the data.sql file.

File: queries.sql [hunk 1]
```diff
@@ -85,3 +85,71 @@ JOIN animals a
...
+LIMIT 1;
+
+SELECT vets.name AS vet_name
+FROM visits
+JOIN vets ON visits.vet_id = vets.id
+JOIN animals ON visits.animal_id = animals.id
+WHERE animals.name = 'Maisy Smith'
+ORDER BY visits.visit_date ASC
+LIMIT 1;
+
```
Comment (Kamba56) [inline queries.sql]: - [x] From your screenshot, I see that this query did not return any result. This should return the first animal visited by Maisy Smith. Kindly correct this, please.

File: queries.sql [hunk 2]
```diff
@@ -85,3 +85,71 @@ JOIN animals a
...
+SELECT species.name AS recommended_specialty
+FROM visits
+JOIN animals ON visits.animal_id = animals.id
+JOIN vets ON visits.vet_id = vets.id
+JOIN specializations ON vets.id = specializations.vet_id
+JOIN species ON specializations.species_id = species.id
+WHERE animals.name = 'Maisy Smith'
+GROUP BY species.name
+ORDER BY COUNT(*) DESC
+LIMIT 1;
```
Comment (Kamba56) [inline queries.sql]: - [x] From your screenshot, this query also did not return any result. Kindly correct the query to return the animal type Maisy Smith should consider specializing in based on the animal type she had the most appointments with.

**Review by Kamba56 — CHANGES_REQUESTED** [review]
Hi @MarcoDDM, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great using JOIN clause to query database!👏👏👏
- [x]

## Required Changes ♻️
_Check the comments under the review._

 - [ ] Kindly consider updating your getting started section. You should include a brief description of how a visitor can set up a local copy of your work.
image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql
Comment (MarcoDDM) [reply queries.sql:1]: But in the requirements there are no mention to that... just the queries and screenshot....

**Review by DeliceLydia — APPROVED** [review]
## Hi @MarcoDDM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed changes



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**